my-app/src/components/CompanyInformationForm.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Input from './Input';
+import Textarea from './Textarea';
+import CodeInput from './CodeInput';
+
+export default class CompanyInformationForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { validationStatus: {} };
+    }
+
+    checkFormValidation = () => {
+        const statusObject = this.state.validationStatus;
+
+        if (Object.keys(statusObject).length !== 3) return false;
+
+        for (const property in statusObject) {
+            if (statusObject[property] === false) return false;
+        }
+        return true;
+    };
```

Comment:
nie lepiej jako ten obiekt z walidacją trzymać tablicę?
coś w stylu:
```js
state = {
  errors: [ ]
}
```
i jeśli errors.length === 0 to wszystko jest ok, a jak nie jest, to dodawać tam jakieś errory:
`errors = [ "TOO_SHORT", "WRONG_NAME" ]` etc?